### ISSUE TITLE
[Feature] Pre-exported assets

### ISSUE THREAD

**Issue by nauvalazhar** [nauvalazhar/code-design#6]
Each design challenge should have a button to download all pre-exported assets, it would be easier for the dev, and no need to export all the assets on their own.

**Comment by perdiDev** [comment]
What do  you think if we put the assets in public directory?

**Comment by nauvalazhar** [comment]
Yeah, it is a good idea. The easier way is to deliver the assets as a zip package for each challenge. The hardest part is exporting all the assets in Figma 🤣 

But, I think we can do it gradually.

**Comment by perdiDev** [comment]
Can you assign this task to me?
